README.md

```diff
@@ -33,12 +33,20 @@ This library is the abstraction of Xendit API for access from applications writt
         -   [Validate OTP for direct debit payment](#validate-otp-for-direct-debit-payment)
         -   [Get direct debit payment by ID](#get-direct-debit-payment-by-id)
         -   [Get direct debit payment by reference ID](#get-direct-debit-payment-by-reference-id)
-    -   [Disbursements](#disbursements)
-        -   [Create Disbursement](#create-disbursement)
-        -   [Create Batch Disbursement](#create-batch-disbursement)
-        -   [Get Disbursement by ID](#get-disbursement-by-id)
-        -   [Get Disbursement by External ID](#get-disbursement-by-external-id)
-        -   [Get Disbursement Available Banks](#get-disbursement-available-banks)
+    -   [IDR Disbursements](#idr-disbursements)
+        -   [Create an IDR Disbursement](#create-an-idr-disbursement)
+        -   [Create an IDR Batch Disbursement](#create-an-idr-batch-disbursement)
+        -   [Get an IDR Disbursement by ID](#get-an-idr-disbursement-by-id)
+        -   [Get an IDR Disbursement by External ID](#get-an-idr-disbursement-by-external-id)
+        -   [Get an IDR Disbursement Available Banks](#get-an-idr-disbursement-available-banks)
+    -   [PHP Disbursements](#php-disbursements)
```

Comment:
@hglohika can we add a bit of wording on these two:

`IDR Disbursements for Indonesia`
`PHP Disbursements for Philippines`

just so they don't confuse the PHP currency against PHP the language

Reply:
Agree

Reply:
@xenditjoshjimenez fff28fd4c7f22fcf5f2a29a582db4086205449f2

Reply:
@xen-HendryZheng Help to review it again. Thank you.